packages/gestalt/src/Avatar.js

```diff
@@ -19,11 +20,28 @@ const Square = (props: *) => (
   </Box>
 );
 
-const DefaultAvatar = ({ name }: { name: string }) => {
+type DefaultAvatarProps = {
+  name: string,
+  useDefaultIcon: boolean,
+};
+
+const DefaultAvatar = ({ name, useDefaultIcon }: DefaultAvatarProps) => {
```

Comment:
@nachofassini inlined the props + used exact objects instead https://flow.org/en/docs/types/objects/#toc-exact-object-types